src/theme/components/template.ts

```diff
@@ -1,113 +1,68 @@
 import { defineStyleConfig } from "@chakra-ui/react";
 import { defineStyle } from "@chakra-ui/system";
-/**
- * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
- */
+
+export const responsiveGap = { base: "1rem", md: "1.5rem", xl: "1rem" };
```

Comment:
is xl supposed to be 2rem?

Reply:
I don't believe so, but @bigfishdesign13, could you please confirm?

Reply:
It looks like the values for `responsiveGap` and `responsivePadding` were flip-flopped. For `responsiveGap`, the `xl` value should be `"2rem"`. And for `responsivePadding`, the `xl` value should be `"1rem"`.